packages/valory/skills/abstract_abci/handlers.py

```diff
@@ -402,7 +402,7 @@ def apply_snapshot_chunk(  # pylint: disable=no-self-use
             performative=AbciMessage.Performative.RESPONSE_APPLY_SNAPSHOT_CHUNK,
             target_message=message,
             result=Result(ResultType.REJECT),
-            refetch_chunks=[],
-            reject_senders=[],
+            refetch_chunks=tuple(),
+            reject_senders=tuple(),
```

Comment:
removes some error messages from the logs

Reply:
Can you point us to the spec where this is defined as tuple rather than list type?

Reply:
It is expected tuples in and returns list when you query this from tendermint / protobuf side.

https://github.com/valory-xyz/open-autonomy/blob/94cf2d80103f8e2513f95630522c87684041c757/packages/valory/protocols/abci/README.md?plain=1#L165-L166

https://github.com/valory-xyz/open-autonomy/blob/94cf2d80103f8e2513f95630522c87684041c757/packages/valory/protocols/abci/message.py#L529-L539

If you run the test on main you'll see the error message in the logs as well

```
/open-autonomy (main)> pytest -s tests/test_connections/test_abci/test_tendermint_encoder.py
...
2022-08-30 11:48:55 [   ERROR] Invalid type for content 'refetch_chunks'. Expected 'tuple'. Found '<class 'list'>'. (message.py:1270)
```